src/StrawberryShake/Client/src/Transport.WebSockets/Protocols/GraphQLWebSocket/GraphQLWebSocketMessageParser.cs

```diff
@@ -110,7 +110,11 @@ private void ParseMessageProperty(ref GraphQLWebSocketMessage message)
             case _i:
                 if (fieldName.SequenceEqual(Id))
                 {
-                    Expect(JsonTokenType.String);
+                    if (_reader.TokenType != JsonTokenType.Null)
```

Comment:
How can the message ID be null?

Reply:
https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md

Reply:
in the protocol its defined that the id is required on messages that have an id. Otherwise no id property is expected.

Reply:
When running a GraphQL server using [GraphQL.AspNet 1.4.2](https://www.nuget.org/packages/GraphQL.AspNet/1.4.2), this issue occurs 100% of the time. However, when using Postman, the subscription works fine.
So, would it be reasonable to consider this a bug in GraphQL.AspNet?

Reply:
I created [issue #168](https://github.com/graphql-aspnet/graphql-aspnet/issues/168) in the [graphql-aspnet](https://github.com/graphql-aspnet/graphql-aspnet) repository.